src/app/public/modules/affix/dom-utils.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  SkyAffixAdapterCoords
+} from './affix-adapter-coords';
+
+/**
+ * Validates coordinates are fully visible within an element.
+ * @param parent
+ * @param coords
+ */
+export function verifyCoordsVisibleWithinElement(
```

Comment:
"verify" seems kind of redundant here. Could we just call this `coordsVisibleWithinElement` ? That way, the `true` or `false` that's returned answers that question.

Reply:
Hm. I prefer function names that are either questions or commands. What about `isOffsetVisibleWithinParent`?

Reply:
Yeah, I like that. It captures the spirit of what I was getting to. When I get a true/false returned from something named `verify`, I'm not really sure what that means. `is` makes a lot more sense. 👍